spec/workers/email_worker_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
```

Comment:
remove the file if you aren't going to add any examples.

Reply:
Done.